src/main/java/io/github/syst3ms/skriptparser/expressions/ExprDateNow.java

```diff
@@ -0,0 +1,62 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.util.SkriptDate;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * The current date, the one from yesterday or the one from tomorrow.
+ *
+ * @name Now
+ * @type EXPRESSION
+ * @pattern (now|yesterday|tomorrow)
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class ExprDateNow implements Expression<SkriptDate> {
+
+	static {
+		Main.getMainRegistration().addExpression(
+				ExprDateNow.class,
+				SkriptDate.class,
+				true,
+				"(0:now|1:yesterday|2:tomorrow)");
+	}
+
+	int parseMark;
+
+	@Override
+	public boolean init(Expression<?>[] expressions, int matchedPattern, ParseContext parseContext) {
+		parseMark = parseContext.getParseMark();
+		return true;
+	}
+
+	@Override
+	public SkriptDate[] getValues(TriggerContext ctx) {
+		switch (parseMark) {
+			case 0:
+				return new SkriptDate[] {SkriptDate.now()};
+			case 1:
+				return new SkriptDate[] {new SkriptDate(SkriptDate.now().getTimestamp() - 1000 * 60 * 60 * 24)};
```

Comment:
I'm aware that everyone and their mother knows how time works so this doesn't look weird to anyone, but it's still a magic number that should be moved to a constant field.